lib/StripeResource.js

```diff
@@ -228,21 +228,31 @@ StripeResource.prototype = {
       return true;
     }
 
-    // Retry on conflict and availability errors.
-    if (res.statusCode === 409 || res.statusCode === 503) {
+    if (res.headers && res.headers['stripe-should-retry'] === 'false') {
```

Comment:
Can we put some docs on this to describe generally what `Stripe-Should-Retry` is?

Reply:
Good idea, will do